_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -145,6 +145,30 @@ For examples of how to define scanner rules for network interfaces, see the
 [`instance_network_interface_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/instance_network_interface_rules.yaml)
 rule file.
 
+## Lien scanner
+Allow customers to ensure Liens for their projects exist and are configured correctly.
+
+For examples of how to define scanner rules for lien, see the
+[`lien_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/lien_rules.yaml)
```

Comment:
This should point to the `master` branch.

Reply:
Should be `stable` instead, we forgot to update that branch for a while since it's never being used so it's now outdated, we can update the documentation later once that branch is up to date.